ui/store/actions.js

```diff
@@ -745,7 +745,7 @@ export function updateEditableParams(txId, editableParams) {
       log.error(error.message);
       throw error;
     }
-
+    await forceUpdateMetamaskState(dispatch);
```

Comment:
[q] why is this necessary?

Reply:
To ensure that transaction state is properly updated before hitting the confirm screen routing logic after clicking the "Next" button